test/slax/projects/worker_test.exs

```diff
@@ -17,12 +17,11 @@ defmodule Slax.Projects.Worker.Test do
           expiration_date: Timex.shift(now, days: 3)
         )
 
-      repo2 =
-        insert(:project_repo,
-          org_name: "org2",
-          repo_name: "repo2",
-          expiration_date: Timex.shift(now, days: 3)
-        )
+      insert(:project_repo,
+        org_name: "org2",
+        repo_name: "repo2",
+        expiration_date: Timex.shift(now, days: 3)
+      )
```

Comment:
Do you still need this?

Reply:
Oh I think I see they're in the string

Reply:
Yeah I have it to add test coverage for multiple repos expiring on the same day